nanovdb/nanovdb/unittest/TestNanoVDB.cu

```diff
@@ -0,0 +1,2528 @@
+// Copyright Contributors to the OpenVDB Project
```

Comment:
I see that `TestNanoVDB.cu` has been added, but I do not see any change in the `unittest/CMakeLists.txt`. Would you mind adding, the following lines at the end of `unittest/CMakeLists.txt`:
```
if(NANOVDB_USE_CUDA)
  add_executable(nanovdb_test_cuda "TestNanoVDB.cu")
  target_link_libraries(nanovdb_test_cuda PRIVATE nanovdb GTest::GTest GTest::Main)
  add_test(nanovdb_cuda_unit_test nanovdb_test_cuda)
endif()
```

Reply:
done